recursive-list-main/src/components/breadcrumbs/breadcrumbs.styles.ts

```diff
@@ -0,0 +1,7 @@
+import { makeStyles } from "@material-ui/core";
+
+export const useBreadcrumbsStyles = makeStyles(() => ({
+  crumbContainer: {
+    marginBottom: "25px",
```

Comment:
check if mui theme spacing can be used here

Reply:
Yes, made changes by using MUI Theme